### PR TITLE
Added `hyperocean` theme

### LINKED ISSUE #60: Theme Awesomeness Check
Love the store and how it looks! Thank you all a lot for your work!
I am using this theme lately and I was thinking if it would be a good fit for the store;
- Link: https://github.com/klauscfhq/hyperocean
- Screenshot::
ocean
If you think it is interesting enough I can create a pull request! Thank you again for your time!

### PR DESCRIPTION
**Pull request by novddd**
Added the `hyperocean` theme by @klauscfhq! Also this pull request fixes #60 😄

### CODE REVIEW

**Comment by novddd** [comment]
Please let me know if I have anything else to do for this pull request! 😄

**Comment by novddd** [comment]
Resolved a minor merge conflict 😄

**Comment by timothyis** [comment]
Thanks! And thanks for the review @klauscfhq!

File: plugins.json [hunk 1]
```diff
@@ -378,5 +378,18 @@
...
+  {
+    "name": "hyperocean",
+    "description": "Deep oceanic blue Hyper theme",
+    "type": "theme",
+    "preview": "https://raw.githubusercontent.com/klauscfhq/hyperocean/master/media/screen.png",
+    "colors": [
+      "#0F111A",
+      "#3A75C4",
+      "#703FAF",
+      "#87D3F8"
```
[the remark this answers is not in the document]
Comment (novddd) [reply plugins.json]: Thank you a lot! Happy to help! 😄

**Comment by novddd** [comment]
No problem at all! 😄

**Pull request merged**